keps/sig-node/127-user-namespaces/README.md

```diff
@@ -803,6 +839,11 @@ logs or events for this purpose.
 
 ###### How can someone using this feature know that it is working for their instance?
 
+Check if any pod has the pod.spec.HostUsers field set to false and is on RUNNING state on a node
+that meets all the requirements.
+
+There are step-by-step examples in the Kubernetes documentation too.
```

Comment:
Link?

Reply:
I can add a link to this: https://kubernetes.io/docs/tasks/configure-pod-container/user-namespaces/

I didn't do it because links here can easily become stale over time.

Reply:
Added